### PR TITLE
Week-1 HTML/CSS

### PR DESCRIPTION
**Pull request by BenzeneAlcohol**

### CODE REVIEW

File: week_1/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* Once you have completed the assignment, submit a Pull Request with your work to the branch `week1_assignment` **(not `main`)**.
+* Your Pull Request will be graded by the automated tests.
+* Status of your grading will be indicated using GitHub tags on your _PR_.
+
+---
+
+## Assignment 1 - Ruby Basics
+
+--
+
```
Comment (hrushikeshj) [inline week_1/README.md:13]: We don't have ruby in the first week this time

File: week_1/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Assignments - Week 1
+
+* Fork this repository and use the `week1_<your_name>` branch for your work.
+* Once you have completed the assignment, submit a Pull Request with your work to the branch `week1_assignment` **(not `main`)**.
+* Your Pull Request will be graded by the automated tests.
```
Comment (mittal-parth) [inline week_1/README.md:5]: No automatic tests for HTML and CSS then, so remove this

File: week_1/HTML.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+The HTML `id` attribute is used to specify a unique id for an HTML element. The important difference between class and an id is that an id has to be unique. The functionality of class and id remains the same.
+
+```
+<div id="city">
+    Mangalore
+</div>
+```
+
+
```
Comment (mittal-parth) [inline week_1/HTML.md:135]: I think we can cover more here.
Introduce all form elements. 
Maybe navbar and footer (unless you explicitly want them to explore these)

File: week_1/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Assignments - Week 1
+
```
Comment (mittal-parth) [inline week_1/README.md:2]: Also add lines that ask them to try these things hands on. Provide links to an online editor as well.

File: week_1/HTML.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
```
Comment (mittal-parth) [inline week_1/HTML.md:136]: Add different types of lists also.

File: week_1/HTML.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+    <h1>Welcome to my first webpage!</h1>
+    <p>This is a paragraph</p>
+    <p>This is the <a href="https://www.iris.nitk.ac.in">link</a> to IRIS NITK.</p>
+    <div class="city">
+        The college is situated in Mangalore.
+    </div>
+</body>
+</html>
+```
+
```
Comment (mittal-parth) [inline week_1/HTML.md:157]: Would be nice if you can add a preview screenshot here

File: week_1/CSS.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+- `font-size`: This attribute specifies the size of the font to be used for the text in an element. It is often specified in pixels or em units.
+- `color`: This attribute specifies the color of the text in an element. It can be specified using a named color, such as "red", or using a hexadecimal code, such as "#FF0000".
+- `background-color`: This attribute specifies the background color of an element. It works in the same way as the color attribute, but it applies to the element's background instead of its text.
+- `border`: This attribute is used to specify the width, style, and color of an element's border. It is often used to add visual boundaries around elements.
+
+These are just a few examples of the many attributes available in CSS. There are many other features and techniques in CSS, including the ability to apply styles based on the state of an element, the use of media queries to apply different styles for different screen sizes and devices, and the use of CSS preprocessors to write more concise and powerful stylesheets. For more information on these and other topics, you can refer to the resources.
+
```
Comment (mittal-parth) [inline week_1/CSS.md:93]: At least one example with HTML and CSS written together with its preview would be nice

**Review by mittal-parth — CHANGES_REQUESTED** [review]

**Comment by BenzeneAlcohol** [comment]
Done, @mittal-parth and @hrushikeshj

**Pull request merged**